lib/ui/painting.dart

```diff
@@ -311,9 +311,9 @@ class Color {
   /// See <https://en.wikipedia.org/wiki/Relative_luminance>.
   double computeLuminance() {
     // See <https://www.w3.org/TR/WCAG20/#relativeluminancedef>
-    final double R = _linearizeColorComponent(red / 0xFF);
-    final double G = _linearizeColorComponent(green / 0xFF);
-    final double B = _linearizeColorComponent(blue / 0xFF);
+    final double R = _linearizeColorComponent(r);
```

Comment:
Now that we have colorspaces, should these be taught to clamp or convert to a non-extended CS as well?

Reply:
I added an assert that extended srgb is in't used.  I don't think it will make sense with that.